docs/specifications/tests/Zone-TP/zone11.md

```diff
@@ -0,0 +1,199 @@
+[Connectivity01]. This test case will just ignore non-responsive name servers or
+name servers not giving a correct DNS response for an authoritative name server.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+## Summary
+
+Message Tag                          | Level   | Arguments            | Message ID for message tag
+:------------------------------------|:--------|:---------------------|:--------------------------------------------
+Z11_SPF1_FOUND_AND_VALID             | INFO    |                      | The *Child Zone* publishes a valid SPF version 1 policy.
+Z11_SPF1_LOOKUP_LIMIT_EXCEEDED       | ERROR   |                      | Evaluating the published SPF version 1 policy would require more than 10 DNS lookups.
+Z11_SPF1_MORE_THAN_ONE_SAME_MODIFIER | ERROR  | spf_modifier         | The published SPF version 1 policy contains the {spf_modifier} modifier more than once.
+Z11_SPF1_MULTIPLE_RECORDS            | ERROR   |                      | The domain publishes more than one SPF version 1 policy.
+Z11_SPF1_REDIRECT_AND_ALL            | WARNING |                      | The published SPF version 1 policy contains both the "all" keyword and a redirect modifier.
+Z11_SPF1_REDIRECT_NOT_AT_END         | WARNING |                      | The published SPF version 1 policy contains a redirect modifier, but it does not appear at the end of the string.
+Z11_SPF1_SYNTAX_ERROR                | ERROR   |                      | The published SPF version 1 policy has a syntax error.
+Z11_SPF_NOT_CONFIGURED               | INFO    |                      | The *Child Zone* does not publish an SPF policy.
+Z11_SPF_WRONG_VERSION                | WARNING | spf_version          | The *Child Zone* publishes an SPF policy with invalid version number {spf_version}.
+
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [argument list].
+
+## Test procedure
+
+In this section and unless otherwise specified below, the term "[DNS Query]"
+follows the specification for DNS queries as specified in [DNS Query and
+Response Defaults]. The handling of the DNS responses on the DNS queries follow,
+unless otherwise specified below, what is specified for [DNS Response] in the
+same specification.
+
+
+1. Create a [DNS Query] with query type TXT query and query name *Child Zone*
+   ("TXT query").
+
+2. Create three empty sets, "TXT-Data", "SPF-Policies" and "SPFv1-Policies".
+
+3. Obtain the set of name server IP addresses using [Method4] and [Method5]
+   ("Name Server IP").
+
+4. For each name server in *Name Server IP* do:
+
+   1. Send *TXT Query* to the name server and collect the [DNS Responses][DNS
+      Response].
```

Comment:
If you send one query, there will only be one response, right?

Reply:
Yes, my bad.